spec/app_spec.rb

```diff
@@ -174,4 +174,17 @@ def test_es_check(service_available, status='green', timed_out=false)
       expect(last_response.body).to include File.expand_path(__FILE__)
     end
   end
+
+  describe 'config' do
+    describe 'Elasticsearch client' do
+      subject { Elasticsearch::Model.client }
```

Comment:
I believe this test is testing that it's reading the application information correctly, which I don't believe the other tests cover.